connect4/gameLogic.ts

```diff
@@ -0,0 +1,150 @@
+import {IMove, deepClone} from '../common/common';
```

Comment:
Commenting needed, I think I got all the idea of the gameLogics since we all know connect4 and it ain't that complex, however I think you should still leave a few commenting.